includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -421,4 +410,91 @@ private function getPartnerSlot() {
 		}
 		return null;
 	}
+
+	private function getLogo() {
+		$logo = [
+			// Deprecated
+			'header' => [
+				'type' => 'link-image',
+				'href' => $this->getHref( 'fandom-logo' ),
+				// 'image' is deprecated use 'image-data' instead
+				'image' => 'wds-company-logo-fandom-powered-by-wikia',
+				'image-data' => [
+					'type' => 'wds-svg',
+					'name' => 'wds-company-logo-fandom-powered-by-wikia',
+				],
+				'title' => [
+					'type' => 'text',
+					'value' => 'Fandom powered by Wikia'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Fandom powered by Wikia" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)